schemas/get-ami-set-response.yml

```diff
@@ -0,0 +1,39 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:                      "Get AMI Set Response"
+description: |
+  An AMI Set from the provisioner
+type:                       object
+properties:
+  id:
+    type: string
+    description: |
+      Name of the AMI set
+  amis:
+    type: array
+    items:
+      type: object
+    description: |
+      AMIs in this set
+    properties:
+      region:
+        type: string
+        description: |
+          The Amazon AWS region of the AMI set
+      hvm:
+        type: string
+        description: |
+          The AMI that uses HVM virtualization type
+      pv:
+        type: string
+        description: |
+          The AMI that uses PV virtualization type
```

Comment:
Oh, indentation is a bit funny here, too -- it says `amis` has properties, rather than the items in `amis`.  My comment was indicating that the region, hvm, and pv properties should be limited and all required.